### ISSUE TITLE
Portal has changed - instructions hard to follow

### ISSUE THREAD

**Issue by andrePKI** [MicrosoftLearning/AZ-301-MicrosoftAzureArchitectDesign#53]
In AZ-301T01_Lab_Mod02_Deploying Service Instances as Components of Overall Azure Solutions.md
Ex 1, Task 3, step 15
The "Deployment Center link in the Code Deployment section" is now called Container settings.

In Ex2, task 2, step 11, the instructions don't match the portal anymore.
Also, the default json template contains ', "parameters": {}' near the end

**Comment by polichtm** [comment]
Would you mind clarifying what you mean by the following?
the default json template contains ', "parameters": {}' near the end
rgds
Marcin

**Comment by andrePKI** [comment]
I mean that the default template from the portal contains that section, whereas the copy in the lab does not.

**Comment by polichtm** [comment]
updated

**Comment by andrePKI** [comment]
@polichtm Marcin, Excellent, thanks!

**Comment by polichtm** [comment]
You're welcome - appreciate the input
